packages/illustrations/vitest.setup.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable import/no-extraneous-dependencies */
+import * as matchers from '@testing-library/jest-dom/matchers'
+import '@testing-library/jest-dom/vitest'
+import { cleanup } from '@testing-library/react'
+import { setup } from '@utils/test'
+import { afterAll, afterEach, beforeAll, expect, vi } from 'vitest'
+
+expect.extend(matchers)
+
+setup()
+
+beforeAll(() => {
```

Comment:
Can't we move this file into `@utils/test`?

Reply:
Yes done !